crop/sample-plan.json

```diff
@@ -0,0 +1,78 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "/crop/sample-plan",
+    "description": "A general plant sampling schema for planned sampling",
+    "allOf": [
+        {
+            "$ref": "sample.json"
+        }
+    ],
+    "properties": {
+        "status": {
+            "enum": [
+                "cancelled",
+                "scheduled",
+                "inProgress",
+                "completed",
+                "paused",
+                "partiallyCompleted"
+            ]
+        },
+        "operation": {
+            "const": "MaturitySample",
+            "type": "string"
+        }
+    },
+    "required": ["status", "operation"],
+    "examples": [
+        {
+            "id": "3772c0e4-f3a6-4fbf-b6cb-38c21316cf83",
+            "identifiers": [
+                {
+                    "scheme": "pureFarmingEntity",
+                    "id": "3772c0e4-f3a6-4fbf-b6cb-38c21316cf83"
+                }
+            ],
+            "name": "Maturity Sampling for vineyard XX on date YY?",
+            "meta": {
+                "sourceId": {
+                    "scheme": "pureFarmingDataSource",
+                    "id": "indevin.api"
+                },
+                "modified": "2021-09-10T12:34:56.000Z"
+            },
+            "links": [
+                {
+                    "relationship": "actual",
+                    "contentType": "vineyard/fruitSampleAnalysis",
+                    "related": "some uri to the actual results of this planned sample?"
+                }
+            ],
+            "observationDate": "2021-02-23T21:00:00.000Z",
+            "holding": {
+                "contentType": "vineyard",
+                "identifier": {
+                    "scheme": "SWNZ",
+                    "id": "NL015"
+                },
+                "name": "Golden Hills",
+                "uri": ""
+            },
+            "feature": {
+                "contentType": "subBlock",
+                "identifier": {
+                    "scheme": "indevinSublockId",
```

Comment:
This bit here needs replacing with something that doesn't reference a customer (along with any associated data that might have come from Indevin).